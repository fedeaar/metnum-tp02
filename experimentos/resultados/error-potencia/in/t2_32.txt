7.355759000000000000e+06 3.133538000000000000e+06 2.412290000000000000e+05 -2.754312000000000000e+06 -1.159660000000000000e+05 -1.170098000000000000e+06 7.405000000000000000e+04 7.291790000000000000e+05 3.129480000000000000e+05 8.907250000000000000e+05 9.933300000000000000e+05 -7.675590000000000000e+05 6.858000000000000000e+05 -8.338250000000000000e+05 1.629260000000000000e+06 6.239390000000000000e+05 -1.554654000000000000e+06 5.314040000000000000e+05 5.718870000000000000e+05 4.546858000000000000e+06
3.133538000000000000e+06 6.361505000000000000e+06 -1.022957000000000000e+06 -1.714025000000000000e+06 -2.713309000000000000e+06 -1.762880000000000000e+06 -3.442920000000000000e+05 1.008748000000000000e+06 4.372690000000000000e+05 1.116647000000000000e+06 1.473815000000000000e+06 -4.966970000000000000e+05 -5.822270000000000000e+05 9.882470000000000000e+05 1.360917000000000000e+06 4.200800000000000000e+05 -2.880999000000000000e+06 8.000360000000000000e+05 5.387750000000000000e+05 2.263637000000000000e+06
2.412290000000000000e+05 -1.022957000000000000e+06 6.775590000000000000e+06 -1.807694000000000000e+06 6.712530000000000000e+05 2.066100000000000000e+05 2.577597000000000000e+06 -1.251108000000000000e+06 1.589142000000000000e+06 -2.272562000000000000e+06 3.166984000000000000e+06 -3.845798000000000000e+06 1.817088000000000000e+06 -1.498054000000000000e+06 1.767661000000000000e+06 1.352056000000000000e+06 -1.523264000000000000e+06 -3.858790000000000000e+05 -9.715020000000000000e+05 2.406785000000000000e+06
-2.754312000000000000e+06 -1.714025000000000000e+06 -1.807694000000000000e+06 6.816926000000000000e+06 6.103300000000000000e+05 -1.539469000000000000e+06 -1.466286000000000000e+06 -2.960553000000000000e+06 -1.244441000000000000e+06 -2.913787000000000000e+06 -3.475333000000000000e+06 7.675290000000000000e+05 -1.514115000000000000e+06 1.014283000000000000e+06 1.577449000000000000e+06 -1.888333000000000000e+06 1.096129000000000000e+06 1.091966000000000000e+06 -1.933351000000000000e+06 -2.833752000000000000e+06
-1.159660000000000000e+05 -2.713309000000000000e+06 6.712530000000000000e+05 6.103300000000000000e+05 6.122715000000000000e+06 1.121713000000000000e+06 1.183681000000000000e+06 -2.970410000000000000e+05 -9.592170000000000000e+05 -7.366200000000000000e+05 4.351290000000000000e+05 -1.652541000000000000e+06 -6.245930000000000000e+05 -2.173442000000000000e+06 -1.313809000000000000e+06 -1.381155000000000000e+06 1.660009000000000000e+06 -2.838670000000000000e+05 -4.151060000000000000e+05 -8.362990000000000000e+05
-1.170098000000000000e+06 -1.762880000000000000e+06 2.066100000000000000e+05 -1.539469000000000000e+06 1.121713000000000000e+06 5.988401000000000000e+06 2.810940000000000000e+05 -6.217210000000000000e+05 7.206000000000000000e+05 1.706953000000000000e+06 2.197080000000000000e+06 1.151239000000000000e+06 8.967120000000000000e+05 -1.286446000000000000e+06 -2.998252000000000000e+06 8.214210000000000000e+05 1.116193000000000000e+06 -3.238900000000000000e+04 3.747860000000000000e+05 -1.073797000000000000e+06
7.405000000000000000e+04 -3.442920000000000000e+05 2.577597000000000000e+06 -1.466286000000000000e+06 1.183681000000000000e+06 2.810940000000000000e+05 3.796634000000000000e+06 8.827000000000000000e+05 1.604962000000000000e+06 -1.708280000000000000e+05 4.278160000000000000e+05 -2.648925000000000000e+06 -1.392750000000000000e+06 -4.695970000000000000e+05 4.697260000000000000e+05 8.351670000000000000e+05 -4.446310000000000000e+05 -2.808580000000000000e+05 -1.991634000000000000e+06 2.101358000000000000e+06
7.291790000000000000e+05 1.008748000000000000e+06 -1.251108000000000000e+06 -2.960553000000000000e+06 -2.970410000000000000e+05 -6.217210000000000000e+05 8.827000000000000000e+05 6.703139000000000000e+06 7.787310000000000000e+05 1.172770000000000000e+05 1.850030000000000000e+05 1.764410000000000000e+05 6.790800000000000000e+05 -1.128980000000000000e+06 -1.268990000000000000e+05 -1.432232000000000000e+06 1.361944000000000000e+06 -5.722750000000000000e+05 -8.982990000000000000e+05 -4.103700000000000000e+04
3.129480000000000000e+05 4.372690000000000000e+05 1.589142000000000000e+06 -1.244441000000000000e+06 -9.592170000000000000e+05 7.206000000000000000e+05 1.604962000000000000e+06 7.787310000000000000e+05 5.649239000000000000e+06 7.400190000000000000e+05 1.799556000000000000e+06 -5.546720000000000000e+05 2.219903000000000000e+06 -2.118702000000000000e+06 1.275690000000000000e+05 1.471933000000000000e+06 -2.771900000000000000e+05 -1.621310000000000000e+06 9.356200000000000000e+05 9.910810000000000000e+05
8.907250000000000000e+05 1.116647000000000000e+06 -2.272562000000000000e+06 -2.913787000000000000e+06 -7.366200000000000000e+05 1.706953000000000000e+06 -1.708280000000000000e+05 1.172770000000000000e+05 7.400190000000000000e+05 7.922379000000000000e+06 -3.077770000000000000e+05 1.235646000000000000e+06 8.740110000000000000e+05 1.897558000000000000e+06 -2.496908000000000000e+06 -1.535809000000000000e+06 -7.472480000000000000e+05 -3.164881000000000000e+06 1.956964000000000000e+06 7.757540000000000000e+05
9.933300000000000000e+05 1.473815000000000000e+06 3.166984000000000000e+06 -3.475333000000000000e+06 4.351290000000000000e+05 2.197080000000000000e+06 4.278160000000000000e+05 1.850030000000000000e+05 1.799556000000000000e+06 -3.077770000000000000e+05 7.338900000000000000e+06 -2.647060000000000000e+06 2.514694000000000000e+06 -1.899936000000000000e+06 -7.481360000000000000e+05 2.816243000000000000e+06 -2.378625000000000000e+06 -1.864620000000000000e+05 2.095532000000000000e+06 5.636150000000000000e+05
-7.675590000000000000e+05 -4.966970000000000000e+05 -3.845798000000000000e+06 7.675290000000000000e+05 -1.652541000000000000e+06 1.151239000000000000e+06 -2.648925000000000000e+06 1.764410000000000000e+05 -5.546720000000000000e+05 1.235646000000000000e+06 -2.647060000000000000e+06 5.936811000000000000e+06 1.296200000000000000e+04 -4.501230000000000000e+05 -2.511322000000000000e+06 1.481000000000000000e+05 1.364552000000000000e+06 -8.120510000000000000e+05 1.912036000000000000e+06 -2.578041000000000000e+06
6.858000000000000000e+05 -5.822270000000000000e+05 1.817088000000000000e+06 -1.514115000000000000e+06 -6.245930000000000000e+05 8.967120000000000000e+05 -1.392750000000000000e+06 6.790800000000000000e+05 2.219903000000000000e+06 8.740110000000000000e+05 2.514694000000000000e+06 1.296200000000000000e+04 8.874459000000000000e+06 -3.899050000000000000e+05 4.742160000000000000e+05 -4.354600000000000000e+05 -1.610056000000000000e+06 -2.257660000000000000e+06 2.627072000000000000e+06 -1.711729000000000000e+06
-8.338250000000000000e+05 9.882470000000000000e+05 -1.498054000000000000e+06 1.014283000000000000e+06 -2.173442000000000000e+06 -1.286446000000000000e+06 -4.695970000000000000e+05 -1.128980000000000000e+06 -2.118702000000000000e+06 1.897558000000000000e+06 -1.899936000000000000e+06 -4.501230000000000000e+05 -3.899050000000000000e+05 6.527826000000000000e+06 2.335800000000000000e+04 -1.841378000000000000e+06 -1.706028000000000000e+06 -3.528850000000000000e+05 -1.461672000000000000e+06 1.097000000000000000e+05
1.629260000000000000e+06 1.360917000000000000e+06 1.767661000000000000e+06 1.577449000000000000e+06 -1.313809000000000000e+06 -2.998252000000000000e+06 4.697260000000000000e+05 -1.268990000000000000e+05 1.275690000000000000e+05 -2.496908000000000000e+06 -7.481360000000000000e+05 -2.511322000000000000e+06 4.742160000000000000e+05 2.335800000000000000e+04 6.627837000000000000e+06 4.228800000000000000e+05 -7.409160000000000000e+05 1.502447000000000000e+06 -2.776743000000000000e+06 1.485526000000000000e+06
6.239390000000000000e+05 4.200800000000000000e+05 1.352056000000000000e+06 -1.888333000000000000e+06 -1.381155000000000000e+06 8.214210000000000000e+05 8.351670000000000000e+05 -1.432232000000000000e+06 1.471933000000000000e+06 -1.535809000000000000e+06 2.816243000000000000e+06 1.481000000000000000e+05 -4.354600000000000000e+05 -1.841378000000000000e+06 4.228800000000000000e+05 8.348053000000000000e+06 -1.223860000000000000e+06 1.070982000000000000e+06 2.001550000000000000e+06 2.295768000000000000e+06
-1.554654000000000000e+06 -2.880999000000000000e+06 -1.523264000000000000e+06 1.096129000000000000e+06 1.660009000000000000e+06 1.116193000000000000e+06 -4.446310000000000000e+05 1.361944000000000000e+06 -2.771900000000000000e+05 -7.472480000000000000e+05 -2.378625000000000000e+06 1.364552000000000000e+06 -1.610056000000000000e+06 -1.706028000000000000e+06 -7.409160000000000000e+05 -1.223860000000000000e+06 5.647005000000000000e+06 1.205557000000000000e+06 -5.941500000000000000e+04 -4.590010000000000000e+05
5.314040000000000000e+05 8.000360000000000000e+05 -3.858790000000000000e+05 1.091966000000000000e+06 -2.838670000000000000e+05 -3.238900000000000000e+04 -2.808580000000000000e+05 -5.722750000000000000e+05 -1.621310000000000000e+06 -3.164881000000000000e+06 -1.864620000000000000e+05 -8.120510000000000000e+05 -2.257660000000000000e+06 -3.528850000000000000e+05 1.502447000000000000e+06 1.070982000000000000e+06 1.205557000000000000e+06 4.610967000000000000e+06 -8.610560000000000000e+05 4.813560000000000000e+05
5.718870000000000000e+05 5.387750000000000000e+05 -9.715020000000000000e+05 -1.933351000000000000e+06 -4.151060000000000000e+05 3.747860000000000000e+05 -1.991634000000000000e+06 -8.982990000000000000e+05 9.356200000000000000e+05 1.956964000000000000e+06 2.095532000000000000e+06 1.912036000000000000e+06 2.627072000000000000e+06 -1.461672000000000000e+06 -2.776743000000000000e+06 2.001550000000000000e+06 -5.941500000000000000e+04 -8.610560000000000000e+05 6.511662000000000000e+06 -1.193577000000000000e+06
4.546858000000000000e+06 2.263637000000000000e+06 2.406785000000000000e+06 -2.833752000000000000e+06 -8.362990000000000000e+05 -1.073797000000000000e+06 2.101358000000000000e+06 -4.103700000000000000e+04 9.910810000000000000e+05 7.757540000000000000e+05 5.636150000000000000e+05 -2.578041000000000000e+06 -1.711729000000000000e+06 1.097000000000000000e+05 1.485526000000000000e+06 2.295768000000000000e+06 -4.590010000000000000e+05 4.813560000000000000e+05 -1.193577000000000000e+06 9.094628000000000000e+06
