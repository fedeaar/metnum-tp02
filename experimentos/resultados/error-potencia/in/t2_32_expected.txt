2.402922926302091777e+07
1.963836397107682005e+07
1.635304479084639624e+07
1.223743161772041954e+07
1.172803339391599409e+07
9.838375354609712958e+06
7.465112214437759481e+06
7.230893519333618693e+06
5.107603239417511970e+06
4.573476742407304235e+06
3.724428586039211135e+06
3.216743029646555427e+06
2.669764139150419272e+06
2.043831957315490814e+06
1.368104218833521241e+06
8.576525304428890813e+05
4.464234190000919625e+05
2.383708764292667329e+05
1.970917859267615422e+05
4.646035042924879963e+04
